packages/api-clients/open-api/catalogApi.yml

```diff
@@ -2151,9 +2151,12 @@ components:
       properties:
         instanceId:
           type: string
-          format: uuid
-          description: Optional identifier to be appended to the template name
-          required: false
+        isSignalHubEnabled:
+          type: boolean
+        isConsumerDelegable:
+          type: boolean
+        isClientAccessDelegable:
+          type: boolean
```

Comment:
Why this changes are necessary in the context of this PR? 
this impacts "create eservice instance by template", these changes could be inconsistent with specifications of [SRS](https://pagopa.atlassian.net/wiki/spaces/PDNDI/pages/1429864566/SRS+Template+e-service#[Work-Item-7]-Creazione-e-gestione-istanza-template-(PIN-6032)). If it's not relevant for model fixes I suggest to create a separate PR with this changes.